GVFS/GVFS/CommandLine/ServiceVerb.cs

```diff
@@ -35,6 +35,13 @@ public class ServiceVerb : GVFSVerb
             HelpText = "Prints a list of all mounted repos")]
         public bool List { get; set; }
 
+        [Option(
+            "log-mount-failure-in-stderr",
+            Default = false,
+            Required = false,
+            HelpText = "This parameter is reserved for internal use.")]
+        public bool RedirectMountFailuresToStderr { get; set; }

```

Comment:
I don't think we need a new flag for this

Reply:
Removed